src/components/Hero.tsx

```diff
@@ -36,7 +36,11 @@ const StyledDiv = styled.div`
   width: 100%;
   height: auto;
   z-index: 1;
-  background: url(${HeroBgDesktop}) no-repeat top center;
+  background: url(${HeroBgDesktop}) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
```

Comment:
i don't think u need all the browser-specific rules, depending on ur build tools, they are added automatically. Check if gatsby provide that in their build or not

Reply:
These are for the parallax effect. Edge specifically had issues without adding these so I had to add them explicitly